graphql/tests/query/version/queries/version.graphql

```diff
@@ -7,6 +7,7 @@
     createTime
     errors
     finishTime
+    generatedTaskCounts
```

Comment:
nit. since `generatedTaskCounts` is a subresolver, I think the test would fit slightly better as a new folder at `graphql/tests/version/generatedTaskCounts`